pkg/cloud/services/loadbalancer/loadbalancer.go

```diff
@@ -305,6 +305,7 @@ func (s *Service) getOrCreateAPILoadBalancer(openStackCluster *infrav1.OpenStack
 		Description: names.GetDescription(clusterResourceName),
 		Provider:    lbProvider,
 		Tags:        openStackCluster.Spec.Tags,
+		APIServerLoadBalancerAZ: openStackCluster.spec.apiSeverLoadBalancerAZ
```

Comment:
I assume this works based on the tags however I assume you have tested to not specify any AZ for your LB just to make sure the CreateLoadBalancer doesn't break if an empty or null value is received

Reply:
Ok. I will check about it.
TY

Reply:
BTW looks like CI is failing due to a missing comma at the end of this line. I'll add a suggestion below:

```suggestion
		APIServerLoadBalancerAZ: openStackCluster.spec.apiSeverLoadBalancerAZ,
```

Reply:
I changed code.